# Generated by Django 3.2.15 on 2023-02-06 20:55

from django.conf import settings
from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    dependencies = [
        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
        ('data_files', '0011_alter_datafile_section'),
        ('search_indexes', '0002_standardized_field_names'),
    ]

    operations = [
        migrations.CreateModel(
            name='ParserError',
            fields=[
                ('id', models.AutoField(primary_key=True, serialize=False)),
                ('row_number', models.IntegerField()),
                ('column_number', models.IntegerField()),
                ('error_message', models.TextField()),
                ('error_type', models.TextField()),
                ('item_number', models.IntegerField()),
                ('field_name', models.TextField()),
                ('file', models.ForeignKey(null=True, on_delete=django.db.models.deletion.CASCADE, related_name='parser_errors', to='data_files.datafile')),
                ('user', models.ForeignKey(null=True, on_delete=django.db.models.deletion.CASCADE, related_name='parser_errors', to=settings.AUTH_USER_MODEL)),
            ],
            options={
                'db_table': 'parser_error',
            },
        ),
    ]
